cypress/e2e/Login.cy.js

```diff
@@ -0,0 +1,81 @@
+import { MEMBERS } from "../fixtures/members";
+
+describe("Login behaviour Test", () => {
+  const { UNKNOWN_EMAIL, UNKNOWN_PASSWORD, WRONG_PASSWORD } = MEMBERS;
+
+  beforeEach(() => {
+    // Include `cy.visit()` command in the beforeEach function
+    // so that it runs before each test to visit the same URL at the start of all the tests
+    cy.visit(Cypress.env("REACT_APP_GRAASP_COMPOSE_HOST_STAGE"));
+    // Perform an assertion with `should` to verify redirection
+    cy.url().should(
+      "be.equal",
+      Cypress.env("REACT_APP_AUTHENTICATION_HOST_STAGE") + "/signin"
+    );
+  });
+
+  it("Sign in with unknown email does not pass", () => {
+    const { email, password, statusCode } = UNKNOWN_EMAIL;
+
+    // Listen for uncaught exceptions and prevent Cypress from failing the test
+    cy.on("uncaught:exception", (err, runnable) => {
+      return false;
+    });
+
+    cy.intercept("POST", "/login-password").as("login");
```

Comment:
Let's already use query-client's exported routes. This comment applies for all endpoint urls.

Reply:
Also, you can create and reuse a Cypress command to intercept the endpoint (avoid code duplication), and/or you could apply the interception in `beforeEach`.

Reply:
Using query-client gives problems due to unmatched dependencies. We should discuss this

Reply:
Fixed code duplication